.circleci/config.yml

```diff
@@ -295,6 +295,33 @@ workflows:
           - sam_deploy_development
           context: [ deployment-development-wcivf, slack-secrets ]
           filters: { branches: { only: [ main, master, staging ] } }
+
+      - sam_deploy:
+          name: sam_deploy_staging
+          dc-environment: staging
+          vpc-id: vpc-69cef901
+          subnet-ids: "subnet-e2b4f198,subnet-5a911a16,subnet-818bb2e8"
+          ssl-certificate-arn: arn:aws:acm:us-east-1:897471774344:certificate/507f9d28-0c00-4886-8446-3ae3e6276e2a
+          instance-type: t3a.medium
+          domain: "wcivf.club"
+          requires:
+          - build_and_test
+          # NB should this be dependent on successful sam_deploy_development job?
```

Comment:
I'm not sure if this is a c&p or a review question, but assuming the latter then the answer is it's a bit of a trade off. By not including it here, we have a dev environment that doesn't get nuked by merges to master. OTOH the dev environment can get a bit stale/out of sync with stage/prod wrt to migrations, config changes etc. This means when you come to use said dev environment it can be a bit more of a pain to get it working. I lean slightly towards having it isolated (i.e. staging is not dependent on a successful dev deploy), but can see both sides.